backend/src/api/PollController.ts

```diff
@@ -105,11 +94,7 @@ export default class PollController {
       this.vote(req, res),
     )
 
-    this.router.get('/poll/:pollId', validate(getSchema), oauth('читать'), (req, res) => this.getPoll(req, res))
-
-    this.router.get('/polls', validate(listSchema), oauth('читать'), (req, res) => this.getPolls(req, res))
-
-    this.router.post('/polls/batch', validate(batchSchema), oauth('читать'), (req, res) => this.getPollsBatch(req, res))
+    this.router.post('/polls', validate(batchSchema), oauth('читать'), (req, res) => this.getPollsBatch(req, res))
```

Comment:
it's better to name it to `poll/get` or something similar, as our oauth scopes are constructed from paths, and it would make sense to have all poll-related endpoints under the `poll` scope